src/objects/aff/zcl_abapgit_json_handler.clas.abap

```diff
@@ -161,15 +159,15 @@ CLASS zcl_abapgit_json_handler IMPLEMENTATION.
       lv_original_language TYPE sy-langu.
 
 
-    lv_iso_language = co_ajson->get_string( '/header/originalLanguage' ).
+    lv_iso_language = co_ajson->get_string( '/header/original_language' ).
```

Comment:
Just for my understanding. We can change it here to the notation with underscore, because we changed the mapping above to `zcl_abapgit_ajson_mapping=>create_to_snake_case( )`, right?

Reply:
Yes, the time of `when` the mapping is actually performed changed. So, on a parsed json data with mapping, the field names are mapped already. Not later, e.g. `to_abap` as before.